lib/config/presets/github/index.ts

```diff
@@ -15,9 +16,15 @@ const http = new GithubHttp();
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  packageTag: string
```

Comment:
See other comment

Reply:
That should be ok now, I also patched Gitlab accordingly